### PR TITLE
New package: tawk.to version 2.2.0

### PR DESCRIPTION
**Pull request by fontajos**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/112577&drop=dogfoodAlpha

### CODE REVIEW

**Comment by fontajos** [comment]
I am not the author of tawk.to nor am I an employee of tawk.to inc. I just use this program in our company and crated a manifest to use it with winget package manager. I thought that maybe other would profit, if a manifest is committed publicly.

If this is ok, I gladly sign the CLA. If not, please close the pull request.

**Comment by Trenly** [comment]
> I am not the author of tawk.to nor am I an employee of tawk.to inc. I just use this program in our company and crated a manifest to use it with winget package manager. I thought that maybe other would profit, if a manifest is committed publicly.
> 
> If this is ok, I gladly sign the CLA. If not, please close the pull request.

Yes, this is fine

**Comment by fontajos** [comment]
@microsoft-github-policy-service agree

File: manifests/t/tawk/to/2.2.0/tawk.to.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: tawk.to
+PackageVersion: 2.2.0
+PackageLocale: en-US
+Publisher: tawk.to, inc.
+PackageName: tawk.to Windows App
+License: tawk.to Terms of Services
+Copyright: Copyright © 2020 tawk.to
+ShortDescription: Windows App to monitor and chat with the visitors on your website, respond to support tickets, organize contacts and create a help center to empower customers to help themselves.
```
Comment (Trenly) [inline manifests/t/tawk/to/2.2.0/tawk.to.locale.en-US.yaml:11]:
```suggestion
Publisher: tawk.to
PublisherUrl: https://www.tawk.to/
PublisherSupportUrl: https://help.tawk.to/
PrivacyUrl: https://www.tawk.to/privacy-policy/
# Author:
PackageName: tawk.to
PackageUrl: https://www.tawk.to/tawk-to-for-windows/
License: Proprietary
LicenseUrl: https://www.tawk.to/terms-of-service/
Copyright: Copyright © 2020 tawk.to
# CopyrightUrl:
ShortDescription: Windows App to monitor and chat with the visitors on your website, respond to support tickets, organize contacts and create a help center to empower customers to help themselves.
# Description:
Moniker: tawk
```
Making these all as one suggestion to make it easy to commit -
* Publisher should match the value written to Control Panel
* PackageName should match the value written to Control Panel
* Added additional metadata

File: manifests/t/tawk/to/2.2.0/tawk.to.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: tawk.to
+PackageVersion: 2.2.0
+Installers:
+- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
+  Architecture: x64
+  InstallerType: nullsoft
+  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC
```
Comment (Trenly) [inline manifests/t/tawk/to/2.2.0/tawk.to.installer.yaml:10]:
```suggestion
- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
  Architecture: x64
  InstallerType: nullsoft
  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC
- InstallerUrl: https://windows.tawk.to/v2/releases/tawk.to-latest.exe
  Architecture: x86
  InstallerType: nullsoft
  InstallerSha256: C602D7A29AC2533904AF0B91801CF58DE698C73E7D211F439522B66554AE9DFC
```
Website says the package is valid for x86 and x64 systems

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
Hi @fontajos,

This package has an odd situation. It installs normally, but doesn't appear to do anything when launched. But this message keeps appearing in the Application log: 
```
Fault bucket 2050985342008795435, type 5
Event Name: BEX64
Response: Not available
Cab Id: 0

Problem signature:
P1: CameraBarcodeScannerPreview.exe
P2: 10.0.19041.546
P3: ec885135
P4: KERNELBASE.dll
P5: 10.0.19041.1202
P6: c9db1934
P7: 000000000010be3e
P8: c0000409
P9: 0000000000000007
P10: 

Attached files:
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER30E4.tmp.dmp
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3133.tmp.WERInternalMetadata.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3163.tmp.xml
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER3184.tmp.csv
\\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WER31B4.tmp.txt

These files may be available here:
\\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_CameraBarcodeSca_f71e02bba2d6c367f92492657fc634476612ff_0998f9d1_46b470c8-779b-4fa0-b5f4-5c82d2a4d327

Analysis symbol: 
Rechecking for solution: 0
Report Id: 356a7144-3385-4101-a331-1e33e0643f7c
Report Status: 268435456
Hashed bucket: 9549ec35ceaa7d05dc76904d4df7a12b
Cab Guid: 0
```

I'm guessing that the `tawk.to` package is calling the `CameraBarcodeScannerPreview` application, and this is failing as the VM has no camera attached. 
- Is this expected?
- Does this package expect the computer to have a camera? This might count as a `Hardware` situation.

**Comment by fontajos** [comment]
> 

Hi @stephengillie 

well... I can't reproduce this behaviour, since all of our notebooks have a camera. Inside a VM that maybe true. However, I think, that this is probably an issue that wont be seen widely in the "real world", since you don't need a chat answering system offering video + voice + screensharing. I also think, that this maybe be an initialization issue of the tawk.to client.

Since I am no employee of tawk.to Inc. I can't du much here.

**Comment by denelon** [comment]
Supporting custom hardware in validation is "Not Planned".
* https://github.com/microsoft/winget-pkgs/issues/199504

**Pull request closed without merging**